view/NFTView.tsx

```diff
@@ -167,17 +141,22 @@ export default function NFTView() {
     });
   };
 
+
   return (
     <>
       <div style={{ display: 'flex' }}>
         <div>
-          <StatusFilter handleChecks={handleChecks} />
+          {isLoading && <p>Loading...</p>}
+          {isError && <p>Error occurred while fetching data</p>}
+          <StatusFilter
+            handleApplyStatusFilterCheck={handleApplyStatusFilterCheck}
+          />
           <br />
           <br />
           <hr />
           <br />
           <br />
-          <PriceFilter handleApplyClick={handleApplyClick} />
+          <PriceFilter handleApplyPriceFilterClick={handleApplyClick} />
```

Comment:
PriceFilter의 props인 handleApplyPriceFilterClick과 동일한 이름으로 함수를 만들어서 사용해주면 더 좋을 것 같아요~
```suggestion
          <PriceFilter handleApplyPriceFilterClick={handleApplyPriceFilterClick} />
```